src/WebUI/Components/DeploymentsComponent.tsx

```diff
@@ -223,7 +223,7 @@ export class DeploymentsComponent extends BaseComponent<IDeploymentsComponentPro
     private static _getHeadingContent(deployment: V1Deployment): JSX.Element {
         return (
             <div>
-                <h3>{deployment.metadata.name}</h3>
+                <span className="heading-title">{deployment.metadata.name}</span>

```

Comment:
accessibility purpose we can make this as h2, later we can adjust the font size, otherwise this does not look like a heading, if we have adjusted this for accessibility then also good.